pyomo/contrib/mindtpy/single_tree.py

```diff
@@ -486,7 +487,8 @@ def handle_lazy_regularization_problem(self, main_mip, main_mip_results, solve_d
             handle_main_optimal(
                 main_mip, solve_data, config, update_bound=False)
         elif main_mip_results.solver.termination_condition in {tc.infeasible, tc.infeasibleOrUnbounded}:
-            config.logger.info('regularization problem infeasible.')
+            config.logger.info(solve_data.log_note_formatter.format(
+                solve_data.mip_iter, 'Reg '+solve_data.regularization_mip_type, 'infeasible'))
             if config.reduce_level_coef:
                 config.level_coef = config.level_coef / 2
```

Comment:
Are we ever using this? SHould we have a test to execute this case?

Reply:
Nope. We are not using this. I think we should have a test, but we need to find a suitable example.

Reply:
Nope. We are not using this. I think we should have a test, but we need to find a suitable example.

Reply:
This is something that we never wrote in a paper and don't know if it works well in practice.  I would be curiou s to see if it works or not.